jsonschema/google/events/cloud/cloudbuild/v1/BuildEventData.json

```diff
@@ -265,18 +257,8 @@
       "format": "date-time"
     },
     "timeout": {
-      "properties": {
-        "seconds": {
-          "type": "integer",
-          "description": "Signed seconds of the span of time. Must be from -315,576,000,000\n to +315,576,000,000 inclusive. Note: these bounds are computed from:\n 60 sec/min * 60 min/hr * 24 hr/day * 365.25 days/year * 10000 years"
-        },
-        "nanos": {
-          "type": "integer",
-          "description": "Signed fractions of a second at nanosecond resolution of the span\n of time. Durations less than one second are represented with a 0\n `seconds` field and a positive or negative `nanos` field. For durations\n of one second or more, a non-zero value for the `nanos` field must be\n of the same sign as the `seconds` field. Must be from -999,999,999\n to +999,999,999 inclusive."
-        }
-      },
       "additionalProperties": true,
-      "type": "object",
+      "type": "string",
```

Comment:
I'm slightly surprised there's no format part here - does this mean there's no way of a JSON schema consumer telling the difference between a duration and a string?

Reply:
We currently don't have it but we can add the formatting in the future.

Filed https://github.com/chrusty/protoc-gen-jsonschema/issues/66